pages/createCard.jsx

```diff
@@ -1,15 +1,122 @@
 import { Navbar } from "../components/Navbar";
 import { Header } from "../components/Header";
+import styled from "styled-components";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+import useFormPersist from "react-hook-form-persist";
+import Popup from "reactjs-popup";
+
+const CreateCard = ({ vocabularies, setVocabularies }) => {
+  const { register, handleSubmit, watch, setValue, reset } = useForm();
+  useFormPersist("form", { watch, setValue });
+  const [newCard, setNewCard] = useState();
+  const [open, setOpen] = useState(false);
+  const closeModal = () => setOpen(false);
```

Comment:
for consistency i'd add a second function for opening. ;) That way the closing function does not feel alone. 
```suggestion
  const closeModal = () => setOpen(false);
  const openModal = () => setOpen(true);
```

Reply:
Probably the most wholesome improvement! Closing function will get its buddy immediately 🙂